### DISCUSSION TITLE
Int field Searching or filtering

### DISCUSSION THREAD

**Discussion by sumeet030990** [prisma/prisma#14443] [Q&A]
I am working on searching and filtering
for field with type string I can filter properly  i.e using keyword like 
'search' or 'contain'

But if my field type is Int then how can I do the searching 
'search' or 'contain' doesnt work in this

let say my field 'minimum_order_quantity' is having Int type 
with values like 123, 125, 500 etc
so when I seach 5
It should return me the row with minimum_order_quantity values 25 and 500 

same as if I search 12
it should return me 123, 125

Can someone please let me know how to achieve this 
```
model dist_centers {
  id                     Int             @id @default(autoincrement())
  name                   String?         @db.VarChar(200)
  zip_code               String?         @db.VarChar(20)
  dc_id                  String?         @db.VarChar(8)
  shipping_day           Int?
  region_id              Int?
  minimum_order_quantity Int?            @default(25)
 
  stores                 stores[]
  master_orders          master_orders[]
  orders                 orders[]
}
```

**Comment by austincrim** [comment]
Duplicate of https://github.com/prisma/prisma/discussions/14413

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
